dev-docs/bidders/trustx2.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: bidder
+title: TRUSTX 2.0
+description: Prebid TRUSTX 2.0 Bidder Adapter
+pbjs: true
+pbs: true
```

Comment:
historically prebid-server can take longer to review and merge.
You can switch `pbs: false` to get this docs PR merged, once the pbjs adapter is released.